webAO/styles/client.css

```diff
@@ -206,16 +206,21 @@
 #client_court_static {
 	position: absolute;
 	height: 100%;
-	width: 100%;
 	top: 0;
 	left: 0;
+	top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
 }
 
 #client_court {
 	position: absolute;
 	height: 100%;
 	top: 0;
 	left: 0;
+	top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
this is a known trick to center stuff in css but it doesn't work in all situations, need to test this